project/plugins.sbt

```diff
@@ -1 +1,5 @@
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.11")
+
+// Required for running on development mode with instrumentation
+resolvers += Resolver.bintrayIvyRepo("kamon-io", "sbt-plugins")
+addSbtPlugin("io.kamon" % "sbt-aspectj-runner-play-2.6" % "1.1.0")
```

Comment:
This is only required for running on development mode due to how the Play SBT plugin works. Users still need to figure out how to add the weaver when running on production. I particularly like the sbt-javaagent plugin cause it plays nice with sbt-native-packager. There are more details [here](http://kamon.io/documentation/1.x/recipes/adding-the-aspectj-weaver/#deploying-with-sbt-native-packager).